### ISSUE TITLE
Does a Technique G219 example for SC 2.5.7 Dragging fail SC 2.5.1 Pointer Gestures?

### ISSUE THREAD

**Issue by melaniephilipp** [w3c/wcag#2248]
In [Sufficient Technique G219](https://www.w3.org/WAI/WCAG22/Techniques/general/G219), the fourth example is: “In a Kanban implementation for process management, tasks can be dragged horizontally from one ‘swimming lane’ to another in order to change the status of tasks. A directional horizontal swipe over an item moves it to the adjacent lane. This process can be repeated until the item is in the desired lane.” 

Does this example fail 2.5.1 Pointer Gestures because the alternate to dragging is a gesture that is a "directional horizontal swipe"? The [Understanding Doc for 2.5.1 Pointer Gestures](https://www.w3.org/WAI/WCAG22/Understanding/pointer-gestures) says: “Examples of path-based gestures include swiping,... dependent on the direction of interaction” 

Should this example be removed from the sufficient technique for 2.5.7 Dragging because swiping fails Pointer Gestures?

**Comment by alastc** [comment]
From a quick read I agree that example should be removed. @detlevhfischer are we missing something?

**Comment by patrickhlauke** [comment]
swiping can be lenient, depending on how precise/path-based/directional the motion needs to be. but for clarity, that example should probably be changed to something like "Each item also provides buttons to move the item to the previous and next adjacent lanes"

**Comment by alastc** [comment]
There's a very similar example to that right above, so I think we could just delete the example raised here.

**Comment by alastc** [comment]
Closed by #2356
